src/renderer/components/widgets/channels/ChannelInput/MentionPoper.tsx

```diff
@@ -42,7 +42,9 @@ export const MentionPoper: React.FC<MentionPoperProps> = ({ anchorEl, children,
   const classes = useStyles({})
```

Comment:
We should rename this component to MentionPopper, to make it clear this has nothing to do with popes.

Reply:
Someone named it that way long time ago and I'm aware of the typo but I didn't change that because I want to avoid imports problems

Reply:
an issue has been created for stuff like this, resolving